src/index.ts

```diff
@@ -24,6 +24,29 @@ export abstract class AlpineComponent {
     [key: string]: any;
 }
 
+export declare type Alpine = {
+	version: string;
+	pauseMutationObserver: boolean;
+	magicProperties: { [name: string]: function };
+	ignoreFocusedForValueBinding: boolean;
+	pauseMutationObserver: boolean;
+	onComponentInitializeds: Array<any>;
+	onBeforeComponentInitializeds: Array<any>;
+	onComponentInitializeds: (callback: function) => void;
```

Comment:
probably replace all the functions here with `CallableFunction`

Reply:
I wasn't aware of that type, done